craco.config.js

```diff
@@ -0,0 +1,18 @@
+const CracoLessPlugin = require('craco-less');
```

Comment:
❓ Why this syntax?

Reply:
Because its just a regular node module, I think this is fine.